pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -323,6 +360,15 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 				format.Pod(pod),
 				err)
 			dswp.desiredStateOfWorld.AddErrorToPod(uniquePodName, err.Error())
+			if strings.Contains(err.Error(), pvcBeingDeletedMsg) {
+				if pvcSource := podVolume.VolumeSource.PersistentVolumeClaim; pvcSource != nil {
+					klog.V(4).Infof("pvc %s/%s is being deleted", pod.Namespace, pvcSource.ClaimName)
+					if dswp.pvcsUnderDeletion == nil {
+						dswp.pvcsUnderDeletion = make(map[string]bool)
```

Comment:
It would be better to initialize this when we initialize dswp

Reply:
It is initialized there.
This is for use by test case.